docs/components/modeler/desktop-modeler/element-templates/defining-templates.md

```diff
@@ -73,6 +77,40 @@ For example, given the following `$schema` definition, the application takes `0.
 
 The JSON schema versioning is backward-compatible, meaning that all versions including or below the current one are supported.
 
+## Execution platform compatibility
+
+You can define template compatibility with different execution platforms using the `engines` property.
+
+This property is a dictionary object, where the execution platform names are the keys, and the [semantic version](https://semver.org/) ranges are the values.
+
+For example, the following `engines` definition specifies that the template is compatible with Camunda 8.6 or higher.
```

Comment:
Given this, should this be backported to 8.6?

Reply:
Addressed with https://github.com/camunda/camunda-docs/pull/4780/commits/a86bcfe7b6696db8ad77da8f452fd4009f62befa